go.mod

```diff
@@ -1,45 +1,45 @@
 module github.com/openshift/api
 
-go 1.20
+go 1.21.3
```

Comment:
We don't want to be that precise. So only `go 1.21` is enough. But for now, I'd stick with golang 1.20, since that will allow you to test this and other PRs with the current builders available.

Reply:
When you're changing this, you'll need to update https://github.com/openshift/api/blob/master/.ci-operator.yaml to match the golang version.

Reply:
>We don't want to be that precise. So only go 1.21 is enough. But for now, I'd stick with golang 1.20, since that will allow you to test this and other PRs with the current builders available.

`go mod tidy` has automatically updated it to `1.21.3`. However, as of now, I updated the go version to `1.20`